crates/pipeline_manager/migrations/V11__service_probe.sql

```diff
@@ -0,0 +1,12 @@
+
+CREATE TABLE IF NOT EXISTS service_probe (
+    id uuid PRIMARY KEY,          -- Unique identifier
+    tenant_id uuid NOT NULL,      -- Tenant the service and thus probe belongs to
+    service_id uuid NOT NULL,     -- Service the probe belongs to
+    success bool NOT NULL,        -- True if probe succeeded, else false
```

Comment:
It's better to keep this a varchar and on the rust side, use an enum of possible values, one of which is Success. This allows us to encode a lot of different error types (e.g. including network unreachable etc).

Reply:
Now we have status and request/response, which achieves this flexibility.